specs/lightclient/beacon-chain.md

```diff
@@ -630,28 +689,10 @@ def process_penalties(state: BeaconState) -> None:
                 state.leak_score[index] -= leak_score_decrease
 
     # Penalty processing
-    if get_current_epoch(state) % EPOCHS_PER_ACTIVATION_EXIT_PERIOD != 0:
```

Comment:
By removing this, we will reset `leak_epoch_counter` to 0 every epoch. Is that intended?

Reply:
ah, right. I'll fix

Reply:
fixed and added test https://github.com/ethereum/eth2.0-specs/pull/2194/commits/56d3a30a054c6ca44dc3d7e5d8ff33378d064897